docs/developing/styleguide.rst

```diff
@@ -2,22 +2,39 @@
 Coding Style Guide
 ******************
 
-Follow `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ (the official style guide for Python).
+Follow `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ (the official style guide for Python). Most PEP8 formatting conventions are enforced in the build by ``pylint`` and/or ``flake8``, so if you do not follow them the build may not pass.
 
 However, for Ion, we limit the lengths of lines to 150 characters, not 80 characters.
 
-Main points:
+Main points
+===========
 
 - Indent using 4 spaces.
 - Use underscores in favor of camel case for all names except the names of classes.
 - Limit the line length of docstrings or comments to 72 characters.
 - Separate top-level functions and class definitions with two blank lines.
 - Separate method definitions inside a class with a single blank line.
-- Use two spaces before inline comment and one space between the pound sign and comment.
+- Use two spaces before inline comments and one space between the pound sign and comment.
 - Use a plugin for your text editor to check for/remind you of PEP8 conventions.
-- Run the ``flake8`` linter periodically.
 - Capitalize and punctuate comments and Git commit messages properly.
 
+What is enforced in the build
+=============================
+
+At the time of this writing, the Travis build runs the following commands:
+
+.. code-block:: sh
+
+        flake8 --max-line-length 150 --exclude=*/migrations/* .
+        pylint --jobs=0 --disable=fixme,broad-except,global-statement,attribute-defined-outside-init intranet/
+        isort --check --recursive intranet
+
+``flake8`` is a PEP8 style checker, ``pylint`` is a linter (but it also enforces some PEP8 conventions), and ``isort``, when called with these options, checks that all imports are sorted alphabetically.
+
+It is recommended that you run all of these locally before opening a pull request (though the Ion developers sometimes skip running the ``pylint`` check locally because it takes a long time to run). All of them are intended to be run from the root directoy of the Git repository.
```

Comment:
```suggestion
It is recommended that you run all of these locally before opening a pull request (though the Ion developers sometimes skip running the ``pylint`` check locally because it takes a long time to run). All of them are intended to be run from the root directory of the Git repository.
```